java/src/jmri/jmrit/operations/trains/JmritOperationsTrainsBundle_en_GB.properties

```diff
@@ -82,6 +82,12 @@ TrainTerminatingStaging = Train ({0}) Terminating into Fiddle Yard ({1})
 
 # build failure messages
 buildErrorCars      = Not enough wagons ({0}) at departure ({1}) to build train ({2})
+buildErrorStagingFullNote = Note that the program requires an empty fiddle yard for a train to arrive on! See OperationPro help "Staging Tracks" for more info.
+buildErrorStagingFull = All fiddle yards at {0} are occupied, reserved, or cannot service the train
+
+buildErrorStagingEmpty = Could not find a departure fiddle yard at {0} that meets the train requirements
```

Comment:
I'm not a native English speaker but my view is that a fiddle yard is the entire yard which can have multiple tracks.

My suggestion is to use the term "fiddle yard track" here since I suppose that only a single track needs to be empty.

Reply:
That's the term that UK operators told me to use for staging tracks.

From the question what is a fiddle yard?

Usually, a fiddle yard is hidden away behind a backscene, or off to the side where it is not in the direct vision of any spectators. Fiddle yards got their name because it is where the layout operator will fiddle with the trains.

Reply:
No one from the UK has reached out to me to change the term "fiddle yard" to "fiddle yard track".  I agree that it makes more sense to add the word track.

Reply:
After doing some research, I've added "track" where necessary. Thanks for bringing it to my attention.